src/components/Markdown.svelte

```diff
@@ -30,6 +32,21 @@
     // eslint-disable-next-line @typescript-eslint/naming-convention
     dompurify.sanitize(marked.parse(content), { SANITIZE_DOM: false });
 
+  function handleMarkdownFileLinkNavigation(event: any) {
```

Comment:
I prefer it when we callback name is describing the action, so in this case it would be `navigateToMarkdownLink`.